account-decoder/src/parse_token.rs

```diff
@@ -246,15 +255,33 @@ impl UiTokenAmount {
     }
 }
 
-pub fn token_amount_to_ui_amount(amount: u64, decimals: u8) -> UiTokenAmount {
-    let amount_decimals = 10_usize
-        .checked_pow(decimals as u32)
-        .map(|dividend| amount as f64 / dividend as f64);
+pub fn token_amount_to_ui_amount(
```

Comment:
I hate to say it, but this one should probably be properly deprecated as well, since it's part of the public interface. I'm not sure our backward compatibility policy makes explicit promises about solana_account_decoder, but it's a crate with heavy downstream use.
I actually thought we might be using this method in SPL, but I guess not!

Reply:
No problem, done!